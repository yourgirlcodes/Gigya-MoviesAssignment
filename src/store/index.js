import { createStore } from "redux";

const initialState = {
  movies: [],
  search: "",
  loading: false
};

const reducer = (state = initialState, action) => {
  return state;
};

const store = createStore(reducer);

export default store;
