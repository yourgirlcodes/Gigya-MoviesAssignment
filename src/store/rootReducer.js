import { combineReducers } from "redux";
import movies from "./reducers/moviesReducer";

export default combineReducers({
  movies
});