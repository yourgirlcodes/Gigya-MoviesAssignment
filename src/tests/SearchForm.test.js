import React from 'react';
import SearchForm from '../components/Filter/NameSearchForm';
import { create } from "react-test-renderer";
import store from "../store";
import { Provider } from "react-redux";


describe("SearchForm component", () => {
    test("Matches the snapshot", () => {
      const form = create(<Provider store={store} ><SearchForm /></Provider>);
      expect(form.toJSON()).toMatchSnapshot();
    })});