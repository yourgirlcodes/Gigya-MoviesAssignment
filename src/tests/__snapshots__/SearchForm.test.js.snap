// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm component Matches the snapshot 1`] = `
<form
  autoComplete="off"
  className="sc-AxjAm cJdsHN"
  noValidate={true}
>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
      htmlFor="standard-basic"
      id="standard-basic-label"
    >
      Enter Keyword
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="standard-basic"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
      />
    </div>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "marginLeft": "5px",
        "marginRight": "5px",
      }
    }
    tabIndex={0}
    type="submit"
  >
    <span
      className="MuiButton-label"
    >
      Submit
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Clear
    </span>
  </button>
</form>
`;
